components/rmrk/Gallery/Search/SearchBar.vue

```diff
@@ -399,7 +401,7 @@ export default class SearchBar extends mixins(
   }
 
   redirectToGalleryPageIfNeed() {
-    if (this.$route.name === 'index') {
```

Comment:
we need to support 'rmrk' 'westend', so checking `$route.name` is not a property way and I use `$route.path` here.